bonita-integration-tests/bonita-integration-tests-client/src/main/java/org/bonitasoft/engine/identity/UserIT.java

```diff
@@ -734,6 +734,25 @@ public void searchUser() throws BonitaException {
 
         deleteUsers(users);
     }
+    
+    @Test
+    public void searchUserSortById() throws BonitaException {
+        final List<User> users = new ArrayList<User>();
+        users.add(getIdentityAPI().createUser("jgrGF[|00", "bpm", "John", "Taylor"));
+        users.add(getIdentityAPI().createUser("user02", "bpm", "Pierre", "Smith"));
+        users.add(getIdentityAPI().createUser("User00", "bpm", "Marie", "Taylor"));
+
+        // order test
+        SearchOptionsBuilder builder = new SearchOptionsBuilder(0, 10);
+        builder.sort(UserSearchDescriptor.ID, Order.ASC);
+        List<User> usersResult = getIdentityAPI().searchUsers(builder.done()).getResult();
+        assertEquals(5, usersResult.size());
+        for (int i = 0; i < users.size(); i++) {
+            assertEquals(users.get(i), usersResult.get(i));
```

Comment:
Your assertion supposes that IDs are generated in an ascending order. If we change implementation to have random IDs, it wont work any longer. That's why I'm not very fond of it...